lib/rbtree.js

```diff
@@ -188,6 +188,7 @@ RBTree.prototype.remove = function(val) {
             } else if (!isRed(node.getChild(!dir))) {
                 var sibling = p.getChild(!lastDir);
                 if (sibling !== null) {
+                    /* jshint -W073 */
```

Comment:
I chose to "ignore" instead of shuffling the code around and inserting `continue` for the sake of readability.